src/common.rs

```diff
@@ -15,6 +17,10 @@ impl fmt::Display for IconType {
 
 #[derive(thiserror::Error, Debug)]
 pub enum MsgBoxError {
+    #[cfg(target_family = "unix")]
```

Comment:
If you divide the definition of error Create like this, you can't use pattern matching across platforms.

```rust
match err {
    MsgBoxError::Create(_) => eprintln!("{} (type: {}, title: \"{}\", content: \"{}\")",
        err, icon_type, title, content)
}
```

Above code only works for Unix variants. 

```rust
match err {
    MsgBoxError::Create => eprintln!("{} (type: {}, title: \"{}\", content: \"{}\")",
        err, icon_type, title, content)
}
```

Above code only works for non-Unix variants. 

Please make the public API consistent across platforms.